src/Ray.BiliBiliTool.Console/appsettings.json

```diff
@@ -15,7 +15,9 @@
     "ChargeComment": "", //充电后留言
     "DayOfReceiveVipPrivilege": 1, //每月几号自动领取会员权益的[-1,31]，-1表示不指定，默认每月1号；0表示不自动领取
     "DayOfExchangeSilver2Coin": 0, //每月几号执行银瓜子兑换硬币[-1,31]，-1表示不指定，默认每月最后一天；-2表示每天；0表示不进行兑换
-    "DevicePlatform": "android" //执行客户端操作时的平台 [ios,android]
+    "DevicePlatform": "android", //执行客户端操作时的平台 [ios,android]
+    "CustomComicId": "27355", //自定义漫画阅读 comic_id，若不清楚含义请勿修改
```

Comment:
Why not using int but string? I saw it is defined as int in option class.

Reply:
Yes, you are right. Just to make sure the value was captured, I added double quotes out of habit.